cypress/support/commands.js

```diff
@@ -64,4 +87,84 @@ Cypress.Commands.add('createFleet', () => {
     cy.get('#rich-validation-field-name').type('test-fleet')
     cy.get('#rich-validation-field-name').should('have.value', 'test-fleet')
     cy.get('.pf-v5-l-stack__item > .pf-v5-c-label-group > .pf-v5-c-label-group__main > .pf-v5-c-label-group__list').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('#textfield-osImage').should('be.visible')
+    cy.get('#textfield-osImage').type(img)
+    cy.get('#textfield-osImage').should('have.value', img)
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-title').should('contain', 'test-fleet')
+    cy.get('.pf-v5-c-description-list__text > .pf-v5-l-flex', { timeout: 50000 }).should('contain', 'Valid')
+})
+Cypress.Commands.add('editFleet', () => {
+    cy.get('#nav-toggle').click()
+    cy.contains('Edge Management').click()
+    cy.contains('Fleets').click()
+    cy.get('[data-label="Name"]').contains('test-fleet')
+    cy.get('.pf-v5-c-table__action > .pf-v5-c-menu-toggle').click()
+    cy.get('[data-label="Name"] > .fctl-resource-link > a > .fctl-resource-link__text').contains('test-fleet')
+    cy.get('.pf-v5-c-table__action > .pf-v5-c-menu-toggle').should('be.visible')
+    cy.get('.pf-v5-c-table__action > .pf-v5-c-menu-toggle').click()
+    cy.get('[data-ouia-component-id="OUIA-Generated-DropdownItem-8"] > .pf-v5-c-menu__item > .pf-v5-c-menu__item-main > .pf-v5-c-menu__item-text').click()
+    cy.get(':nth-child(2) > .pf-v5-c-form__group-label > .pf-v5-c-form__label > .pf-v5-c-form__label-text').should('contain', 'Fleet name')
+    cy.get('.pf-v5-c-form__group-control > .pf-v5-c-label-group > .pf-v5-c-label-group__main > .pf-v5-c-label-group__list > .pf-v5-c-label-group__list-item > .pf-v5-c-button').click()
```

Comment:
And I think we should avoid using such chained selectors, these can definitely break by small changes in the UI.

In the UI, we can add properties such as "data-testid" which makes explicit that the selector is being used for QE purposes so if we need to make changes to it, we can know in advance that it may break some QE test. 

```suggestion
    [...straightforward and reusable way to access this element].click()
```